mender-convert-package

```diff
@@ -204,8 +205,14 @@ fi
 # way, and write the rootfs image first.
 run_and_log_cmd "mv --force work/rootfs/data work/data"
 
-# Clear this area as it will be contained in the data.img
-sudo rm -rf work/rootfs/data/*
+if [[ "${MENDER_CLIENT_INSTALL}" == y ]]; then
+    # If the client version is less than 3.4, then log a warning
+    if ! miminum_required_version "3.4.0" "${MENDER_CLIENT_VERSION}"; then
```

Comment:
This will also evaluate to false with `master` or `latest`, right? In both cases the warning should not appear

(for `latest` as per today it should appear but by the time we release this code then it should not - so to keep it simple I think we can ignore already)

Reply:
haha, I knew you were gonna comment on this :p 

See https://github.com/mendersoftware/mender-convert/pull/468/files#diff-76ffda63fe1086e8e1289ee58c126e145c92c0bd8a5165f138eb4deb65055674R20 ;)

Reply:
The latest part is a good point. But don't you think it's fine like this, until we release?

Reply:
> haha, I knew you were gonna comment on this :p
> 

Sorry that I didn't read the comment :see_no_evil: 



> The latest part is a good point. But don't you think it's fine like this, until we release?

Yes, that is what I meant - it is okey